src/block_cache/block_cache.c

```diff
@@ -257,7 +268,7 @@ __blkcache_eviction_thread(void *arg)
                     blkcache->removals++;
 
                     WT_STAT_CONN_INCR(session, block_cache_blocks_evicted);
-                    WT_STAT_CONN_DECRV(session, block_cache_bytes, blkcache_item->data_size);
+                    WT_STAT_CONN_DECRV(session, block_cache_bytes, data_size);
```

Comment:
Is this stat only changed from one thread? Perhaps an atomic version has to be used here.

Same question for other stats in this file.

Reply:
There is one but on one seems to care: WT-14224.

Reply:
That's disappointing. I'll chat to you offline about this.

Reply:
What was the outcome here?